specs/capella/fork.md

```diff
@@ -112,9 +112,7 @@ def upgrade_to_capella(pre: bellatrix.BeaconState) -> BeaconState:
         # Execution-layer
         latest_execution_payload_header=pre.latest_execution_payload_header,
         # Withdrawals
-        withdrawal_queue=[],
-        next_withdrawal_index=WithdrawalIndex(0),
-        next_withdrawal_validator_index=ValidatorIndex(0),
+        last_withdrawal_validator_index=ValidatorIndex(0),
```

Comment:
naming-wise, I think we could just keep `next_withdrawal_index` and use this symbol instead of `last_withdrawal_validator_index`

Reply:
I didn't understand this one, I added back now `next_withdrawal_index` as a counter that just simply counts withdrawals since the fork. `last_withdrawal_validator_index` can't be recovered from this and is needed to know from where to start sweeping.

Reply:
yeah I got confused on the naming -- this works!